Finish first batch of getting started tutorial changes (#10579)
wagtail/wagtail: docs/getting_started/index.md

# Getting started

```{toctree}
---
maxdepth: 1
---
tutorial
quick_install
demo_site
integrating_into_django
the_zen_of_wagtail
```

> Wagtail CMS is an open-source content management system that is built on [Django](https://www.djangoproject.com/), a popular Python web framework. It has gained popularity among developers and content editors for its powerful features and intuitive interface, providing a streamlined editing experience. Wagtail offers a comprehensive toolkit for content creation and management, including a rich text editor with formatting options, image and document management, version control, and content scheduling.

Reviewer: ```suggestion
Wagtail is an open-source CMS c(ontent management system) that is built on [Django](https://www.djangoproject.com/), a popular Python web framework. It has gained popularity among developers and content editors for its powerful features and intuitive interface, providing a streamlined editing experience. Wagtail offers a comprehensive toolkit for content creation and management, including a rich text editor with formatting options, image and document management, version control, workflows and content scheduling.
```

Reviewer 2: suggestion to your suggestion: `CMS c(ontent management system)` -> `CMS (content management system)`

Reviewer 3: Suggestion: use the full term and put the abbreviation in parentheses, then use the abbreviation from that point on.

```suggestion
Wagtail is an open-source content management system (CMS) that is built on [Django](https://www.djangoproject.com/), a popular Python web framework. It has gained popularity among developers and content editors for its powerful features and intuitive interface, providing a streamlined editing experience. Wagtail offers a comprehensive toolkit for content creation and management, including a rich text editor with formatting options, image and document management, version control, workflows and content scheduling.
```

Reviewer: Thanks, sorry about my added typo :) 

Agree with Sage's suggestion of the full words and then abbreviation after, all subsequent content can use the abbreviation in this page.

> Developers appreciate Wagtail's highly customizable and modular architecture, which includes built-in support for Django's app structure. This allows them to easily create and integrate custom functionality, making Wagtail suitable for projects of any size. Wagtail excels in handling complex content structures, offering features like hierarchical page organization, robust search capabilities, and content localization.

Author: @lb- The issue regarding whether to stick to the UK spelling style or American was raised at the Core meeting. The majority of the people present at the meeting agreed that we should gradually move away from the UK spelling style, starting with our Getting Started tutorial. The major reason why we agreed to this is that more people globally are used to the American spelling style than the UK spelling style.

> Developers appreciate Wagtail's highly customizable and modular architecture, which includes built-in support for Django's app structure. This allows them to easily create and integrate custom functionality, making Wagtail suitable for projects of any size. Wagtail excels in handling complex content structures, offering features like hierarchical page organization, robust search capabilities, and content localization.

Reviewer: Ok. I do recall seeing something about this, our current docs say we should not spell things this way so we should probably update our own documentation to reflect this decision. 

https://docs.wagtail.org/en/stable/contributing/general_guidelines.html#language still says British English. 

I'd also recommend we then update the docs spell checker, and maybe a bulk update. 

Maybe this spelling change needs its own issue created so there can be a proper plan in place. 

Even as recently as last month I updated some spelling and PRs to British spelling.

> Developers appreciate Wagtail's highly customizable and modular architecture, which includes built-in support for Django's app structure. This allows them to easily create and integrate custom functionality, making Wagtail suitable for projects of any size. Wagtail excels in handling complex content structures, offering features like hierarchical page organization, robust search capabilities, and content localization.

Author: I will add this to my todo list and discuss the way forward with my mentors @thibaudcolas and @vossisboss

> Developers appreciate Wagtail's highly customizable and modular architecture, which includes built-in support for Django's app structure. This allows them to easily create and integrate custom functionality, making Wagtail suitable for projects of any size. Wagtail excels in handling complex content structures, offering features like hierarchical page organization, robust search capabilities, and content localization.

Reviewer 4: I’ve created #10608 so we have a clearer place where this decision is documented and we can plan the change.

> Wagtail stands out as the preferred choice for tens of thousands of organizations globally, including renowned names like Google, NASA, and the British NHS. It has proven scalability, capable of handling high volumes of traffic from millions of visitors every month. What sets Wagtail apart is its ability to extend beyond traditional content management, providing seamless integration with data tools and rich data visualizations.

Reviewer: ```suggestion
Wagtail stands out as the preferred choice for tens of thousands of organisations globally, including renowned names like Google, NASA and the British NHS. It has proven scalability, capable of handling high volumes of traffic from millions of visitors every month. What sets Wagtail apart is its ability to extend beyond traditional content management, providing seamless integration with data tools and rich data visualisations.
```

For now we mostly avoid Oxford commas.

Author: The Google developer documentation style guide strongly advises the use of [serial comma or an Oxford comma](https://developers.google.com/style/commas).

Reviewer: Ok great. No push back from me - I think we should update our writing guides to reflect this preference though. 

We often get lots of PRs changing things either to add or remove this comma, I just want us to have a preference.

Reviewer: We should add a note above the section on Latin phrases. 

https://docs.wagtail.org/en/stable/contributing/general_guidelines.html#latin-phrases-and-abbreviations

Author: @lb- @thibaudcolas @vossisboss I would like your insights in regards to writing and creating content indicating the change from British spelling style to American, as well as our choice of documentation writing style guide. In my opinion, this is more related to documentation guidelines and should be included in [Documentation guidelines] (https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#) or [Writing documentation] (https://docs.wagtail.org/en/stable/contributing/documentation_modes.html).

The reason why I suggest these two sections is that I believe documentation contributors will naturally look at them before any other section regarding documentation guidelines.

Author: @lb- @thibaudcolas @vossisboss Never mind! I can see now that @thibaudcolas already created an issue regarding this. Happy to work on the issue after this pull request is approved and merged.

> For more information about Wagtail and the guiding principles for building websites with it, read [Zen of Wagtail](the_zen_of_wagtail).

Reviewer 4: ```suggestion
For more information about Wagtail and the guiding principles for building websites with it, read [the Zen of Wagtail](the_zen_of_wagtail).
```
